src/styles/blocks/button.scss

```diff
@@ -0,0 +1,23 @@
+.button {
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+
+  &__buy {
+    text-decoration: none;
+    padding: 11px 69px;
+    color: #fff;
+    background-color: $button-color;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border-radius: 5px;
+    border: 1px solid transparent;
```

Comment:
U can add `box-sizing: border-box` to include the border width in the overall width and you don't have to add a transparent border